src/test/crimson/perf_async_msgr.cc

```diff
@@ -31,9 +31,16 @@ struct Server {
   DummyAuthClientServer dummy_auth;
   unique_ptr<Messenger> msgr;
   struct ServerDispatcher : Dispatcher {
-    ServerDispatcher(CephContext* cct)
-      : Dispatcher(cct)
-    {}
+    unsigned msg_len = 0;
+    bufferlist msg_data;
+
+    ServerDispatcher(CephContext* cct, unsigned msg_len)
+      : Dispatcher(cct), msg_len(msg_len)
+    {
+      bufferptr ptr(msg_len);
+      memset(ptr.c_str(), 0, msg_len);
```

Comment:
could just put
```c++
ptr.zero();
```

Reply:
I used `append_zero()` as suggested by @rzarzynski 

But seems the code of this PR is out of sync with https://github.com/cyx1231st/ceph/commits/wip-seastar-msgr-perf2, weird..